SmRtAPI/SmRtAPI/Messages/WordSubMessage.cs

```diff
@@ -17,20 +19,56 @@ public class WordSubMessage : BaseMessage
         /// End time
         /// </summary>
         public double end_time;
-
         /// <summary>
         /// Whether the element represents an end of sentence marker (e.g. full stop, question mark, exclamation mark)
         /// </summary>
         public bool is_eos;
-
         /// <summary>
         /// Whether this punctuation mark attaches to previous or next token.
         /// </summary>
         public string? attaches_to;
```

Comment:
So this could be nullable but we didn't add the json unmarshalling tag `[JsonProperty(NullValueHandling = NullValueHandling.Ignore)]` . Was it on purpose? Given that this version is released I am happy to do it on a following MR rather than here

Reply:
I didn't want to touch the released code, it would change the results people got - probably for the better, by omitting the field, but better to leave alone.